packages/ui-components/src/components/UIFlexibleTable/UIFlexibleTableRow.tsx

```diff
@@ -134,7 +142,11 @@ export function renderTitleRow<T>(props: UIFlexibleTableProps<T>, paddingRight:
                 column.className
             ]);
             rowCellsData.push(
-                <div key={`title-cell-${key}-${i}`} className={className} title={column.tooltip}>
+                <div
+                    key={`title-cell-${key}-${i}`}
+                    className={className}
+                    title={column.tooltip}
+                    id={`columnKey-${key}`}>
```

Comment:
could be issue with unique ids. If we have two tables with different table ids, but columns has same keys, then we will get duplicate ids. I think in such case usually table id is used to build child ids with concatenation

Reply:
fixed